PluginCode/src/main/java/smr/cs/ualberta/libcomp/DatabaseAccess.java

```diff
@@ -500,11 +554,19 @@ public void updateFeedback(ReplacementFeedback replacementFeedback) throws IOExc
         obj.put("action_date", df.format(replacementFeedback.getAction_date()));
 
         String JsonString = obj.toString();
-        String InsertUrllink = "https://smr.cs.ualberta.ca/comparelibraries/api/pluginfeedback/";
+        if (cloudSelect)
+        {
+            if (sendFeedback(JsonString, tokenValue)) {}
```

Comment:
what is the point of this empty if statement?

Reply:
removed